pkg/scheduler/metrics/metrics.go

```diff
@@ -25,23 +25,26 @@ import (
 
 const schedulerSubsystem = "scheduler"
 
+const BindingType = "binding"
+const SchedulingAlgorithmType ="scheduling_algorithm"
+const E2eSchedulingType = "e2e_scheduling"
+
+type resettableCollector interface {
+        prometheus.Collector
+}
+
 // All the histogram based metrics have 1ms as size for the smallest bucket.
 var (
-	E2eSchedulingLatency = prometheus.NewHistogram(
-		prometheus.HistogramOpts{
-			Subsystem: schedulerSubsystem,
-			Name:      "e2e_scheduling_latency_microseconds",
-			Help:      "E2e scheduling latency (scheduling algorithm + binding)",
-			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
-		},
-	)
-	SchedulingAlgorithmLatency = prometheus.NewHistogram(
-		prometheus.HistogramOpts{
+	OperationLatency = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
 			Subsystem: schedulerSubsystem,
-			Name:      "scheduling_algorithm_latency_microseconds",
-			Help:      "Scheduling algorithm latency",
-			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
+			Name: "operation_latency_microseconds",
+			Help: "operation latency",
+			// Make the sliding window of 5h.
+			// TODO: The value for this should be based on our SLI definition (medium term).
```

Comment:
Can you change:
our -> some
medium -> long

Reply:
done